src/Sulu/Bundle/ContentBundle/Twig/ContentTwigExtension.php

```diff
@@ -138,7 +138,10 @@ public function needsAddButtonFunction(PropertyInterface $property)
      */
     public function isMultipleTest($property)
     {
-        return $property->getMinOccurs() > 1;
+        return
+            !is_null($property->getMaxOccurs()) ||
+            $property->getMinOccurs() > 1 ||
+            $property->getMinOccurs() < $property->getMaxOccurs();
```

Comment:
What exactly should this return at all? If you have `minOccurs` as 0 and `maxOccurs` as 1 it would already return true, although it is not really multiple.